src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import metricsReducer from './metrics/metrics';
+
+const reducer = combineReducers({ metricsReducer });
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger, thunk),
```

Comment:
- [ ] It will be good to remove the logger middleware on the deployed project, to ensure the security of data.